docker/Dockerfile

```diff
@@ -46,6 +47,7 @@ RUN chmod o+r /usr/local/bundle/gems/robots-*/lib/robots.rb
 RUN adduser -g msfconsole -D $MSF_USER
 
 RUN /usr/sbin/setcap cap_net_raw,cap_net_bind_service=+eip $(which ruby)
+RUN /usr/sbin/setcap cap_net_raw,cap_net_bind_service=+eip /usr/bin/nmap
```

Comment:
@ilyaglow according to https://secwiki.org/w/Running_nmap_as_an_unprivileged_user `cap_net_admin` is also needed. I think we should also set this on the ruby line above. Can you please add it to the two lines?